README.md

```diff
@@ -18,6 +18,29 @@ Code documentation is automatically generated using Doxygen tool, available at [
 
 ## Build
 
+Clone this repository:
```

Comment:
Shouldn't we add this information in the build guides, rather than main README of the no-OS repository?

Reply:
Yeah we should add a section in the build guides with this info